### ISSUE TITLE
paymentMethodOrder in PaymentElement

### ISSUE THREAD

**Issue by Andrea951112** [flutter-stripe/flutter_stripe#1862]
It is possible to add the paymentMethodOrder parameter to the PaymentElement object?

here the Stripe docs for this feature https://docs.stripe.com/js/elements_object/create_payment_element#payment_element_create-options-paymentMethodOrder

**Comment by remonh87** [comment]
this is available in flutter_stripe_web 6.0.0

**Comment by Andrea951112** [comment]
@remonh87  sorry, but i do not see this implementation in flutter_stripe_web 6.0.0, is still not possible to configure the paymentMethodOrder in the PaymentELement. The PaymentElementOptions are created using only the "layout" parameter.

**Comment by remonh87** [comment]
it is here: https://github.com/flutter-stripe/flutter_stripe/blob/716dbdfd0581da10c2f1ba1d4c0cffd0d16de5d2/packages/stripe_js/lib/src/api/elements/payment_element_options.dart#L23

**Comment by Andrea951112** [comment]
yes, is present in the options object, but is missing the personalization in the PaymentElement object 

`  js.PaymentElementOptions elementOptions() {
    return js.PaymentElementOptions(layout: widget.layout);
  }
  `

**Comment by remonh87** [comment]
yes you are correct I added it to the options

**Comment by remonh87** [comment]
fixed as part of release 6.1.0
